### PR TITLE
Update future work

### PR DESCRIPTION
**Pull request by HDinger**

### CODE REVIEW

**Comment by MrSerth** [comment]
We should use `Schul-Cloud` everywhere in order to comply with the marketing rules.

File: wiki/future-work.md [hunk 1]
```diff
@@ -6,16 +6,17 @@ At this point, concepts will be discussed which could not be implemented due to
 
 There are two applications where automated jobs are tendering:
 
-- Update of external calendar feeds
+- Change of an external calendar feeds
 - Check for waiting notifications
 
 The central Schulcloud-Service will be in charge to update external feeds. For this we offer routes that can be used to modify feeds. The Schulcloud-Service can forward feed changes (delete, edit, add) directly to the calendar service.
+The general concept will look like the following: The Calendar-Service offers one route for each action that needs to be performed on a subscription (for details look at the API specification). These routes can be called periodically by the central Schulcloud-Service. This handling is advantageous for the architecture, because it is possible to run the Cron-Job in a separate docker container. So multiple instances of the Calender-Service can be started without having the problem of handling the according Cron-Jobs directly. Since the Schulcloud shall be used for whole Germany it is likely that there will be multiple instances of the services. With the help of a load balancer it is possible to handle all instances fairly and synchronously.
```
Comment (MrSerth) [inline wiki/future-work.md]: The Docker part is fine.

Comment (MrSerth) [inline wiki/future-work.md]: Which actions on subscriptions do you think of? Are there really multiple ones to be called by a cron job?

File: wiki/future-work.md [hunk 2]
```diff
@@ -6,16 +6,17 @@ At this point, concepts will be discussed which could not be implemented due to
...
 
-To send notifications to the Notification-Service in time, the Calendar-Service periodically checks the database for pending notifications. The time interval is set to five minutes. If a notification occurs during this period, it is forwarded accordingly.
+There is a Notifications-Services which notifies users about upcoming events as well as other information from the Schulcloud. To enable this in time, the Calendar-Service periodically checks the database for pending notifications. The time interval is set to five minutes. If a notification occurs during this period, it is forwarded accordingly. From our point of view it is not necessary to set the interval smaller (or even sent notifications in real time) since there are no urgent notifications about events.
```
Comment (MrSerth) [inline wiki/future-work.md]: Do we have that already? Isn't this action called by the central Schul-Cloud service and thus the interval can be flexible (e.g. parsed as GET query parameter)?

File: wiki/future-work.md (hunk 1 above)
Comment (HDinger) [inline wiki/future-work.md]: You are right. I thought of every action possible on a subscription, but a cron job should do only updates..

**Pull request merged**